### ISSUE TITLE
Full-text search phrase queries support

### ISSUE THREAD

**Issue by LiahimRatman** [qdrant/qdrant#2955]
Hi!

Have seen some plans in Roadmap for exact phrase queries support in full-text search, are this plans still valid btw?
Just wanted to say that this addition will be very useful for cases when you are searching something like "lion king" and you don't want to see results like "the king of narnia was a lion". Currently, the only way to filter this cases is to include the whole text field in response and filter it manually.

All the best!

**Comment by agourlay** [comment]
Have a look at our documentation for full text match https://qdrant.tech/documentation/concepts/filtering/#full-text-match :)

**Comment by generall** [comment]
Hi, @LiahimRatman! We don't yet have phrase matching, but it is something we would like to eventually have indeed

**Comment by cwinsnes** [comment]
Is there any updates on the status of full text phrase queries?

**Comment by cwinsnes** [comment]
@generall Is there any update on the phrase matching? I have found that to be something that would be rather useful to have while still having access to an index.

**Comment by zicklag** [comment]
Phrase search appears to be supported now!

https://qdrant.tech/documentation/concepts/indexing/#phrase-search

**Comment by timvisee** [comment]
Correct! This has been released as part of [Qdrant 1.15](https://github.com/qdrant/qdrant/releases/tag/v1.15.0).
